src/AutoRest.CSharp/Mgmt/AutoRest/MgmtTarget.cs

```diff
@@ -85,6 +85,22 @@ public static void Execute(GeneratedCodeWorkspace project, CodeModel codeModel,
                 project.AddGeneratedFile($"{name}.cs", codeWriter.ToString());
             }
 
+
+            // Write long running operations as individual files.
+            // todo: levarage data-plane code to simplify
+            foreach (var operationGroup in codeModel.OperationGroups.Where(og => og.IsResource(configuration.MgmtConfiguration)))
```

Comment:
Can we follow the pattern of creating a collection of LRO objects in the output library and then looping on that here?

Reply:
Resolved by #1239 and #1242